src/js-marketing-notification.ts

```diff
@@ -1,32 +1,36 @@
 import { LocalStorageManager } from './github/gtmsportswear/js-local-storage-manager@1.0.2/local-storage-manager';
 
+export interface NotificationNode {
+  classNames?: Array<string>,
```

Comment:
Wondering if this needs to be a property, or could it just be applied to the node itself directly before being passed in? It seems like that could simplify the logic here a bit.